pysph/sph/wc/kernel_correction.py

```diff
@@ -225,18 +230,19 @@ def __init__(self, dest, sources, dim=2, tol=0.1):
         super(MixedGradientCorrection, self).__init__(dest, sources)
```

Comment:
In MixedKernelCorrectionPreStep docs, the last equation should be ".. math:" instead of "..math".

Reply:
Thanks!